### PR TITLE
Add optional body parameter to schema

### PR DESCRIPTION
**Pull request by bseverino**
**Description:** Adds body parameter to schema in order to allow for POST, PUT, and PATCH endpoints.

**JIRA:** [MST-203](https://openedx.atlassian.net/browse/MST-203)

Example of what body parameter will output:

Screen Shot 2020-10-05 at 10 50 29 AM

**Merge checklist:**
- [x] All reviewers approved
- [x] CI build is green
- [x] Version bumped
- [x] Changelog record added
- [x] Documentation updated (not only docstrings)
- [x] Commits are squashed

**Post merge:**
- [ ] Create a tag
- [ ] Check new version is pushed to PyPI after tag-triggered build is 
      finished.
- [ ] Delete working branch (if not needed anymore)

**Author concerns:** List any concerns about this PR - inelegant 
solutions, hacks, quick-and-dirty implementations, concerns about 
migrations, etc.

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -11,7 +11,7 @@ deps =
     django22: Django>=2.2,<2.3
     django30: Django>=3.0,<3.1
     drf39: djangorestframework<3.10
-    drflatest: djangorestframework
+    drflatest: djangorestframework<3.12
```
Comment (schenedx) [inline tox.ini:14]: Instead of marking this as the `drflatest`, can you create a `drf311` and use that in the `envlist` array instead?

**Review by schenedx — APPROVED** [review]
One suggestion only.

File: CHANGELOG.rst [hunk 2]
```diff
@@ -14,6 +14,12 @@ Change Log
 Unreleased
 ----------
 
+1.3.3 --- 2020-10-05
+____________________
+
+* Adding option to include a body parameter in requests.
```
Comment (nedbat) [inline CHANGELOG.rst:20]: Since we are extending the interface of the library, shouldn't this have been version 1.4?

Comment (bseverino) [reply CHANGELOG.rst:20]: @nedbat You're right, I can update it - I haven't merged my PR for Registrar yet, so it shouldn't be an issue.

**Pull request merged**